interface/forms/dictation/view.php

```diff
@@ -1,31 +1,60 @@
-<!-- Forms generated from formsWiz -->
 <?php
+/** 
+ *  Dictation Form Edit Saved Data
+ * 
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ * @copyright Copyright (c) 2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * 
+ */
 
-
-
-
+use OpenEMR\Core\Header;
 include_once("../../globals.php");
+
 $returnurl = 'encounter_top.php';
 ?>
-<html><head>
-<?php html_header_show();?>
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+<html>
+<head>
+<title>Review</title>
+   <?php Header::setupHeader(); ?>
+   <script type="text/javascript" src="<?php echo $webroot."/public/assets/ckeditor-4-7-0/ckeditor.js" ?>"</script>
+   <script src="<?php echo $webroot."/public/assets/ckeditor-4-7-0/js/sample.js" ?>" type="text/javascript"></script>
+   <link href="<?php echo $webroot."/public/assets/ckeditor-4-7-0/css/samples.css"; ?>" rel="stylesheet" type="text/css" />
 </head>
 <body class="body_top">
+<div class="container">
 <?php
 include_once("$srcdir/api.inc");
-$obj = formFetch("form_dictation", $_GET["id"]);
+$id = filter_input(INPUT_GET, "id", FILTER_VALIDATE_INT);
+$obj = formFetch("form_dictation", $id);
 ?>
-<form method=post action="<?php echo $rootdir?>/forms/dictation/save.php?mode=update&id=<?php echo attr($_GET["id"]);?>" name="my_form">
-<span class="title"><?php echo xlt('Speech Dictation'); ?></span><Br><br>
-<span class=text><?php echo xlt('Dictation: '); ?></span><br><textarea cols=80 rows=24 wrap=virtual name="dictation" ><?php echo text($obj{"dictation"});?></textarea><br>
-<span class=text><?php echo xlt('Additional Notes: '); ?></span><br><textarea cols=80 rows=8 wrap=virtual name="additional_notes" ><?php echo text($obj{"additional_notes"});?></textarea><br>
-<br>
-<a href="javascript:top.restoreSession();document.my_form.submit();" class="link_submit">[<?php echo xlt('Save'); ?>]</a>
-<br>
-<a href="<?php echo "$rootdir/patient_file/encounter/$returnurl";?>" class="link"
- onclick="top.restoreSession()">[<?php echo xlt('Don\'t Save Changes'); ?>]</a>
-</form>
+   <form method=post action="<?php echo $rootdir?>/forms/dictation/save.php?mode=update&id=<?php echo attr($id);?>" name="my_form">
+     <div class="page-header">
+      <h1><?php echo xlt('Speech Dictation Review/Edit'); ?></h1>
```

Comment:
Probably doesn't makes sense to make this title different on the forms since most forms do not allow this (goal is keeping this uniform), since usually view.php redirects to new.php on a lot of the forms. Also good to keep the title in head the same as this title which should be the same as the form name. So would use 'Speech Dictation' in all those place.